pkg/es/writer/esBulkHandler.go

```diff
@@ -318,3 +319,56 @@ func PostBulkErrorResponse(ctx *fasthttp.RequestCtx) {
 	responsebody["status"] = 400
 	utils.WriteJsonResponse(ctx, responsebody)
 }
+
+func ProcessDeleteIndex(ctx *fasthttp.RequestCtx, myid uint64) {
+	inIndexName := utils.ExtractParamAsString(ctx.UserValue("indexName"))
+
+	convertedIndexNames, indicesNotFound := deleteIndex(inIndexName, myid)
+
+	if indicesNotFound == len(convertedIndexNames) {
+		ctx.SetStatusCode(fasthttp.StatusNotFound)
+		response := make(map[string]interface{})
+		final := make(map[string]interface{})
+		var items = make([]interface{}, 0)
+		items = append(items, *utils.NewDeleteIndexErrorResponseInfo(inIndexName))
+		final["root_cause"] = items
+		final["error"] = *utils.NewDeleteIndexErrorResponseInfo(inIndexName)
+		final["status"] = 404
+		response["error"] = final
+		utils.WriteJsonResponse(ctx, response)
+	} else {
```

Comment:
Is there an HTTP response code for a partial success? If so we should send that when some indexes get deleted but others do not